library/panic_unwind/src/gcc.rs

```diff
@@ -109,5 +109,5 @@ pub unsafe fn cleanup(ptr: *mut u8) -> Box<dyn Any + Send> {
 // determine whether the exception was thrown by their own runtime.
 fn rust_exception_class() -> uw::_Unwind_Exception_Class {
     // M O Z \0  R U S T -- vendor, language
-    0x4d4f5a_00_52555354
+    0x4d4f_5a00_5255_5354
```

Comment:
The `_` separation here seems both purposeful and useful as the comment makes clear. Changing this feels worse to me.

Reply:
agree that and restored